src/main/scala/com/fulcrumgenomics/bam/EstimatePoolingFractions.scala

```diff
@@ -190,7 +189,7 @@ class EstimatePoolingFractions
     iterator.setIncludeNonPfReads(false)
     iterator.setMappingQualityScoreCutoff(this.minMappingQuality)
     iterator.setQualityScoreCutoff(this.minBaseQuality)
-    iterator.toIterator
+    javaIteratorAsScalaIterator(iterator)
   }
```

Comment:
That's a good point, we could have our own SamLocusIterator case class that extends only Iterator.